pkg/importer/avro/transformer.arrai

```diff
@@ -0,0 +1,106 @@
+let util = //{./util};
+
+# Extra items transformed to Sysl type !enum
+let rec extraEnums = \schema
+  cond schema {
+    {'type': (s: 'enum'), 'name': name, ...}:
+      {schema},
+    {'type': (s: 'record'), 'name': name, "fields": (a : fields), ...}:
+      //rel.union(fields => extraEnums(.@item("type"))),
+    {'type': type, 'items': items, ...}: # Map or array
+      cond items {
+        (a: types): //rel.union(types => extraEnums(.@item)), # array
+        (:s): extraEnums(items.s), # primitive
```

Comment:
Nit: The `(:s)` is binding `items.s` to a variable named `s`, so you can just use that.
```suggestion
        (:s): extraEnums(s), # primitive
```